tests/Avalonia.Base.UnitTests/Interactivity/InteractiveTests.cs

```diff
@@ -398,6 +400,28 @@ public void Removing_Control_In_Handler_Should_Not_Stop_Event()
 
             Assert.Equal(new[] { "3", "2b", "1" }, invoked);
         }
+        
+        [Fact]
+        public void Get_Inner_EventArgs_Exist()
+        {
+            var event1 = new KeyEventArgs();
+            var event2 = new SpinEventArgs(SpinDirection.Increase) { Inner = event1 };
+            var root = new RoutedEventArgs() { Inner = event2 };
+            var inner1 = root.GetInnerEventArgs<KeyEventArgs>();
+            Assert.NotNull(inner1);
+            var inner2 = root.GetInnerEventArgs<SpinEventArgs>();
+            Assert.NotNull(inner2);
+        }
+        
+        [Fact]
+        public void Get_Inner_EventArgs_Not_Exist()
```

Comment:
```suggestion
        public void Get_Inner_EventArgs_Does_Not_Exist()
```

Reply:
I have committed this fix. I will push after we have a conclusion on repeatbutton issue. (to save ci resource)